src/chatClient.cpp

```diff
@@ -192,6 +192,14 @@ void Client::heartbeat()
         KR_LOG_WARNING("Heartbeat timer tick without being connected");
         return;
     }
+
+#ifndef KARERE_DISABLE_WEBRTC
+    if (rtc->isCallInProgress())
```

Comment:
I think it's better to add this checkup to `presenced::Client::autoAwayInEffect()`, in order to simply avoid the transition to the _away_ status during a call.